EasyCommands/Common/Primitives.cs

```diff
@@ -110,11 +112,12 @@ public static List<Return> GetTypes(Type type) => type != typeof(object) ? NewLi
         public static Color CastColor(Primitive p) => Cast<Color>(p);
         public static KeyedList CastList(Primitive p) => Cast<KeyedList>(p);
 
-        public static Color? GetColor(String s) => (s.StartsWith("#") && s.Length == 7) ?
-            new Color(HexToInt(s.Substring(1, 2)), HexToInt(s.Substring(3, 2)), HexToInt(s.Substring(5, 2))) :
-            (colors.ContainsKey(s.ToLower()) ? colors[s.ToLower()] : (Color?)null);
+         static Color? GetColor(String s) =>
+            (s.StartsWith("#") && s.Length == 7)
+            ? new Color(HexToInt(s.Substring(1, 2)), HexToInt(s.Substring(3, 2)), HexToInt(s.Substring(5, 2)))
+            : (colors.ContainsKey(s.ToLower()) ? colors[s.ToLower()] : (Color?)null);
 
-        public static Vector3D? GetVector(String s) {
+        static Vector3D? GetVector(String s) {
```

Comment:
nit: you can leave the "public" on any methods/fields global to "Program", as MDK minimize will remove the public modifier.

Reply:
good to know